Big Update:

MatthewSandford opened the pull request:
- Added new biomes, removed old. Most old ones are still there just updated.
- Added new world examples and removed old.
- Modifications to some prefabs to remove dangerous blocks.
- Added decoration and substrate custom nodes. Mostly so that example biomes work.
- Lots of files have changed file location which will cause problems for older configs. This is generally a big issue with our files. Not much we can do here. 
- Added more images to use with the image node.
- New premade 3d cave nodes.
- Hopefully most if not all files are already migrated to v15.

MatthewSandford committed:
Big Update:

Added new biomes, removed old.
Added new world examples and removed old.
Modifications to some prefabs to remove danagerous blocks.
Added decoration and substrate custom nodes. Mostly so that example biomes work.
Lots of files have changed file location which will cause problems for older configs.
Added more images to use with the image node.
New premode caves.
Hopefully most if not all files are already migrated to v15.

nevir commented:
Holy hell you've been busy!  Awesome 👍

nevir commented:
When trying to open either of the new release worlds, the WB consistently crashes with:

https://gist.github.com/nevir/282d0059d34601f0b29984df82004dd2

(I'm guessing there are some features still unreleased via 187 w/ the WB?)

nevir commented:
> * Lots of files have changed file location which will cause problems for older configs. This is generally a big issue with our files. Not much we can do here.

Maybe identify configs by uuid in the future, or similar

nevir commented:
TODO on this one:

* [ ] Move the world configs to a separate branch, until we get the next WB update
* [ ] Migrate any stragglers to v15

MatthewSandford committed:
Update:

-Added world configs for blake.
-Biomes configs added for use in world configs.
-Partly migrated configs.

MatthewSandford committed:
Updated last world for blake

MatthewSandford committed:
Mostly prefab edits.

MatthewSandford committed:
Corruption changes

MatthewSandford committed:
New world configs.

MatthewSandford committed:
Migrate to 16

nevir commented:
Mind if I close this since I failed to update it, and you've already merged all the goodies in?
